tools/logger/convert.c

```diff
@@ -198,6 +198,12 @@ static void process_params(struct proc_ldc_entry *pe,
 		if (p[1] == '%') {
 			/* Skip "%%" */
 			p += 2;
+		} else if (p[1] == 's') {
+			/* check for string printing, because it leads to logger crash */
+			log_err("String printing is not supported\n");
+			pe->params[i] = 0;
```

Comment:
I'd actually like to see something like "<String @ 0x924046F9>" or something, to distinguish between compile time strings and runtime strings. Not mandatory but nice to have.

Reply:
Added